src/functions/getCssValue.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I suggest moving this file to a `utils` directory as we do it in other repos. What do you think?

Reply:
We agreed in the core meeting to rename it.